### PR TITLE
Adding section about access to Azure Storage Explorer

### PR DESCRIPTION
**Pull request by hjaltedaniel-umbraco**
I want to add a section about how users can use Azure Storage Explorer to connect to their blob storage, in case you want to upload media files manually.

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for adding this article @hjaltedaniel-umbraco !

I'll make sure it's reviewed as soon as possible 💪

File: Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Connect to Azure Storage Explorer to upload files manually
+
+:::note
+**Important**: All Umbraco 8 projects created after May 6th 2020 will have Azure Blob Storage by default. That means that this guide only applies to your project has been created after May 6th 2020.
+:::
+
+In case you want to manually upload files to the blob storage provided to your site, you can take advantage of the software "Microsoft Azure Storage Explorer".
+
+:::note
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md:13]: Should we make this a warning instead? Or is that perhaps a bit too much?

```suggestion
:::warning
```

File: Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+**Important**: If you upload your media files manually using this method, they will not be available in the backoffice.
+:::
+
+## Getting the credentials
+The first thing to sort out, if you want to connect to the blob storage of your site is the credentials of your site. You can find the credentials in KUDU (Power Tools) - under the "Environment" section.
+In the environment section, we want to locate the "Environment Variables" and find the following three variables: `APPSETTING_Umbraco.Cloud.StorageProviders.AzureBlob.ContainerName`, `APPSETTING_Umbraco.Cloud.StorageProviders.AzureBlob.Endpoint` and `APPSETTING_Umbraco.Cloud.StorageProviders.AzureBlob.SharedAccessSignature`. Note these down, as we will use them in the upcoming steps.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md:19]:
```suggestion
## Getting the credentials
The first thing to sort out, if you want to connect to the blob storage of your site is the credentials of your site. You can find the credentials in KUDU (Power Tools) - under the "Environment" section.

In the environment section, we want to locate the "Environment Variables" and find the following three variables: 

* `APPSETTING_Umbraco.Cloud.StorageProviders.AzureBlob.ContainerName`
* `APPSETTING_Umbraco.Cloud.StorageProviders.AzureBlob.Endpoint` and 
* `APPSETTING_Umbraco.Cloud.StorageProviders.AzureBlob.SharedAccessSignature`. 

Note these down, as we will use them in the upcoming steps.
```

File: Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+1. Click the "Open connect dialogue" button to get the connect dialogue
+
+    ![Connect my machine](images/storage-explorer-connection.png)
+
+2. Select "Use a shared access signature (SAS) URI" in the first prompt
+    
+    ![Use a shared access signature (SAS) URI](images/select-connection.png)
+
+3. In the URI field, input the information you have gathered earlier in the following format `[Endpoint][ContainerName][SharedAccessSignature]` like the following `https://ucmediastoragewelive.blob.core.windows.net/0e6ee123-5q22-1234-8618-ae7d0043710f?sv=2017-04-17&sr=c&si=umbraco&sig=f84%2F%2FRPPirgdzn15a%2BA12345678901%2FXA%3D&spr=https`
+
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md:36]: I cannot add it here, but could you add the sample URL here in a **code block** instead? 😇 

```suggestion
3. In the URI field, input the information you have gathered earlier in the following format `[Endpoint][ContainerName][SharedAccessSignature]` like the following example:

   `https://ucmediastoragewelive.blob.core.windows.net/0e6ee123-5q22-1234-8618-ae7d0043710f?sv=2017-04-17&sr=c&si=umbraco&sig=f84%2F%2FRPPirgdzn15a%2BA12345678901%2FXA%3D&spr=https`

```

**Review by sofietoft — CHANGES_REQUESTED** [review]
This is awesome @hjaltedaniel-umbraco ! Thanks.

I've made a few suggestions to it - please take a look when you have time.

Also, from where are we linking to this new article?

File: Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md (hunk 1 above)
Comment (hjaltedaniel-umbraco) [reply Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md:13]: I believe it's fine with a warning - that stands out, and also ensures that there are not two notes right after each other 😅

File: Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md (hunk 3 above)
Comment (hjaltedaniel-umbraco) [reply Umbraco-Cloud/Set-Up/Media/Connect-to-Azure-Storage-Explorer/index.md]: Done 👍

**Comment by hjaltedaniel-umbraco** [comment]
Thanks for the feedback @sofietoft 👍 
I've applied the requested changes. As to where we link to the article, I'm not sure. Maybe this knowledge could come in handy when migrating a site? Maybe it should just stay in the sidebar menu - and appear when people who want to know whether Cloud supports "Azure Storage Explorer" search for that. I'm not sure 🤷‍♂️

**Comment by sofietoft** [comment]
Great, thanks for applying the suggestions @hjaltedaniel-umbraco ! 💪 

Where it is right now, it will not be part of the menu, I'm afraid 🤔 
Perhaps we should add a link from the Media article as a starting point?
We will need to update both articles anyways, once Azure Blob Storage becomes the default for ALL Umbraco Cloud projects.

**Comment by hjaltedaniel-umbraco** [comment]
@sofietoft - that's fine, I assume it'll still show up in search results?
I've added a link in the media article in the latest commit.

**Review by sofietoft — APPROVED** [review]
Looks great! 💪

**Pull request merged**